server/src/Application/HumanResource/Leave/Command/DeleteLeaveRequestCommandHandler.spec.ts

```diff
@@ -3,14 +3,14 @@ import { DeleteLeaveRequestCommandHandler } from './DeleteLeaveRequestCommandHan
 import { DeleteLeaveRequestCommand } from './DeleteLeaveRequestCommand';
 import { User } from 'src/Domain/HumanResource/User/User.entity';
 import { LeaveRequestNotFoundException } from 'src/Domain/HumanResource/Leave/Exception/LeaveRequestNotFoundException';
-import { CanLeaveRequestBeRemoved } from 'src/Domain/HumanResource/Leave/Specification/CanLeaveRequestBeRemoved';
+import { DoesLeaveRequestBelongsToUser } from 'src/Domain/HumanResource/Leave/Specification/DoesLeaveRequestBelongsToUser';
```

Comment:
(Si t'as la foi) En anglais le "belong" doit ici être à l'infinitif

```suggestion
import { DoesLeaveRequestBelongToUser } from 'src/Domain/HumanResource/Leave/Specification/DoesLeaveRequestBelongsToUser';
```

Reply:
@florimondmanca done, plus clean effectivement